rclcpp/include/rclcpp/node.hpp

```diff
@@ -139,24 +139,84 @@ class Node : public std::enable_shared_from_this<Node>
   const std::vector<rclcpp::callback_group::CallbackGroup::WeakPtr> &
   get_callback_groups() const;
 
-  /// Create and return a Publisher.
+  /// Create and return a Publisher. Note: This constructor is deprecated
```

Comment:
Why not use the [deprecated](http://www.doxygen.nl/manual/commands.html#cmddeprecated) tag?

Reply:
https://github.com/ros2/rclcpp/pull/673 is the right place for that discussion - I am not opposed but I am concerned that it introduces warnings to existing core code, causing yellow unstable builds across the board, which may not be the desired thing at this time